src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validappeal/ValidateAppealAboutToSubmitHandler.java

```diff
@@ -2,34 +2,67 @@
+import static uk.gov.hmcts.reform.sscs.service.CaseCodeService.generateIssueCode;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
 import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
 import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Appeal;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Benefit;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CaseManagementLocation;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DirectionType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DynamicList;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DynamicListItem;
 import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.FormType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.InterlocReviewState;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsType;
 import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
 import uk.gov.hmcts.reform.sscs.ccd.presubmit.ResponseEventsAboutToSubmit;
-import uk.gov.hmcts.reform.sscs.service.ServiceRequestExecutor;
+import uk.gov.hmcts.reform.sscs.ccd.validation.sscscasedata.AppealPostcodeHelper;
+import uk.gov.hmcts.reform.sscs.exception.CaseManagementLocationService;
+import uk.gov.hmcts.reform.sscs.helper.SscsDataHelper;
+import uk.gov.hmcts.reform.sscs.service.DwpAddressLookupService;
 
 @Component
 @Slf4j
 public class ValidateAppealAboutToSubmitHandler extends ResponseEventsAboutToSubmit implements PreSubmitCallbackHandler<SscsCaseData> {
 
-    private final ServiceRequestExecutor serviceRequestExecutor;
-    private final String bulkScanEndpoint;
+    private final SyaAppealValidator appealValidator;
+    private final SscsDataHelper sscsDataHelper;
+    private final DwpAddressLookupService dwpAddressLookupService;
+    private final AppealPostcodeHelper appealPostcodeHelper;
+    private final CaseManagementLocationService caseManagementLocationService;
+    private final boolean caseAccessManagementFeature;
+
+    public ValidateAppealAboutToSubmitHandler(SyaAppealValidator appealValidator,
+                                              AppealPostcodeHelper appealPostcodeHelper,
+                                              SscsDataHelper sscsDataHelper,
+                                              DwpAddressLookupService dwpAddressLookupService,
+                                              CaseManagementLocationService caseManagementLocationService,
+                                              @Value("${feature.case-access-management.enabled}") boolean caseAccessManagementFeature) { ////check if feature toggle exists, if does remove.
```

Comment:
can comment be removed yet?

Reply:
will remove feature flag and comment as the feature is already enabled